orte/tools/orte-info/components.c

```diff
@@ -56,7 +58,7 @@ OBJ_CLASS_INSTANCE(orte_info_component_map_t,
                    component_map_construct,
                    component_map_destruct);
 
-opal_pointer_array_t component_map;
+opal_pointer_array_t component_map = {{0}};
```

Comment:
arguably `component_map` should be prefixed with `orte_`, but that doesn't make this initialization any less valid